docker/README.md

```diff
@@ -118,7 +118,7 @@ This example uses [the included Docker Compose project](./docker-compose.yml) to
     ZITI_IDENTITY_JSON="$(< /tmp/my-ziti-id.json)" docker-compose run ziti-host run-host --verbose=4
     ```
 
-1. Access the demo server via your OpenZiti Network: [http://hello-docker.ziti](http://hello-docker.ziti)
+1. Access the demo server via your OpenZiti Network: `http://hello-docker.ziti`
```

Comment:
this was clickable before if ziti was running and it's not now. i think it's better as a link?

Reply:
It has potential value as a clickable link, but that requires the reader to know they need to have it running for the link to work, it will get detected as a broken link by the link checker bot when it's imported to the docs site.